### PR TITLE
Next Py3 pytest fix: absolute imports for acs4, render_template

### PR DESCRIPTION
**Pull request by cclauss**
Closes #

### CODE REVIEW

File: openlibrary/plugins/upstream/borrow.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 from infogami import config
 from infogami.utils import delegate
-from infogami.utils.view import public
+from infogami.utils.view import public, render_template
 from infogami.infobase.utils import parse_datetime
 
 from utils import render_template
```
Comment (tfmorris) [inline openlibrary/plugins/upstream/borrow.py]: Should this `render_template` import still be included? Seems like a name conflict...

Comment (cclauss) [reply openlibrary/plugins/upstream/borrow.py]: Did the second commit fix that or do you still see an issue.

Comment (tfmorris) [reply openlibrary/plugins/upstream/borrow.py]: Yup, looks better now. I'll review the rest of the branch.

File: openlibrary/solr/update_work.py [hunk 2]
```diff
@@ -12,15 +11,16 @@
 import simplejson as json
 import six
 from six.moves import urllib
+from six.moves.http_client import HTTPConnection
 import web
 from lxml.etree import tostring, Element, SubElement
 
-from data_provider import get_data_provider
 from infogami.infobase.client import ClientException
```
Comment (tfmorris) [inline openlibrary/solr/update_work.py]: `import ClientException` unused

File: openlibrary/solr/update_work.py [hunk 3]
```diff
@@ -12,15 +11,16 @@
...
 from openlibrary import config
 from openlibrary.catalog.utils.query import set_query_host, base_url as get_ol_base_url
```
Comment (tfmorris) [inline openlibrary/solr/update_work.py]: import `base_url as get_ol_base_url` unused

File: openlibrary/solr/update_work.py [hunk 4]
```diff
@@ -12,15 +11,16 @@
...
 from openlibrary.core import helpers as h
 from openlibrary.core import ia
```
Comment (tfmorris) [inline openlibrary/solr/update_work.py]: `import ia` unused

File: openlibrary/plugins/upstream/borrow.py [hunk 5]
```diff
@@ -12,24 +12,21 @@
...
+from infogami.utils.view import public, render_template
 from infogami.infobase.utils import parse_datetime
 
-from utils import render_template
-
 from openlibrary.core import stats
 from openlibrary.core import msgbroker
 from openlibrary.core import lending
 from openlibrary.core import waitinglist
 from openlibrary.core import ab
```
Comment (tfmorris) [inline openlibrary/plugins/upstream/borrow.py]: `import ab` unused

**Comment by cclauss** [comment]
Please revert https://github.com/internetarchive/openlibrary/pull/2941/commits/1ee9928d497f457050c06bb79909e7ef9608e24f which makes changes unrelated to the stated purpose of the PR and unnecessarily modifies 118 files in vendor/infogami.

**Comment by tfmorris** [comment]
> Are the changes to vendor/infogami deliberate?

No, they're not. I've removed the commit.

**Comment by tfmorris** [comment]
I've resolved all my comments about removing unused imports and rebased the branch to clean up the merge commits.

@hornc I think this is ready to merge (and thanks for catching the unintended infogami changes.)

**Comment by hornc** [comment]
Thanks @cclauss , and @tfmorris for the rebase. This is a lot cleaner now, merging!

**Pull request merged**